index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+    <title>PortfolioS</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <section class="container">
+        <div class="logo">
+            My Logo
+        </div>
+            <div class="button">
+               <img src="button.png" alt="buttom" width="40px" height="40px"> 
+            </div>
+    </section>
+    <section class="second-container">
+      <div class="header">
+        <h1 class=" bigger-text">I'm Albert. Glad to see you!</h1>
+      </div> 
+      <div>
+        <p class="smaller-text">I'am a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      </div> 
+      <div class="connect">
+        LET'S CONNECT
+      </div>
+      <div class="social">
+        <img src="twitter.png" alt="twitter" class="twitter" width="19.99px" height="19.99px">
+        <img src="Linkedin icon@2x.png" alt="linkedin"class="linkedin" wdth="20px" height="20px">
+        <img src="media.png" alt="media" class="media" width="20px" height="16px">
+        <img src="Vector@2x.png" alt="vector" class="vector" width="17px" height="20px">
+        <img src="vecto@2x.png" alt="vecto" class="vecto" width="14px" height="20px">
+      </div>
+    </section>
```

Comment:
- Kindly make sure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). :+1: 
   * Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.